bogons_sync-ripe.sh

```diff
@@ -0,0 +1,250 @@
+#!/usr/bin/env bash
+
+### USER CONFIGURABLE DATA
+
+# Put here any custom ipv4 prefixes you want to append to the output files
+# The format must be an array, e.g:
+# MY_IPV4_PREFIXES=("0.0.0.0/8" "10.0.0.0/8")
+# If you don't have any prefixes of your own, please leave the array declared
+# and empty
+
+MY_IPV4_PREFIXES=()
+
+
+# Put here any custom ipv6 prefixes you want to append to the output files
+# The format must be an array, e.g:
+# MY_IPV6_PREFIXES=("::/8" "0100::/64" "2001:2::/48")
+# If you don't have any prefixes of your own, please leave the array declared
+# and empty
+
+MY_IPV6_PREFIXES=()
+
```

Comment:
i insist on this comment here:

### You may not need to touch anything else below ###

Reply:
Discussed and addressed